content/news/025/index.md

```diff
@@ -317,6 +317,27 @@ Read the [dev blog][triplehex-blog] for more info, follow
 [triplehex-twitter]: https://twitter.com/triplehexdev
 [shattersong-discord]: https://discord.gg/K5RHxVEK6F
 
+### Fish Fight
+
+![Screenshot of fishfight.org website](fishfight-website.png)
+
+[As promised](https://fishfight.itch.io/ff/devlog/281554/fish-fight-reloaded),
+after months of private prototyping of early invitees, the Fish Fight devs are
+finally ready to make their code and community channels available to the
+general public. They also launched their website!
+
+- [Announcement post][fish-announce]
+- [GitHub repo][fish-github] (MIT/Apache)
+- [Discord server][fish-discord]
+- [Design doc][fish-designdoc]
+- [fishfight.org][fish-website]
+
+[fish-announce]: https://fishfight.itch.io/ff/devlog/291737/fish-fight-is-open-source
```

Comment:
Is this meant to go into the August newsletter? The announcement was clearly in September.

Reply:
It’s intended for this one, yeah. By the time the next newsletter comes out this will be old news and we’ll have new September updates to share anyhow, so I’d appreciate it if we could soften the requirements a bit here ☺️

Reply:
Fair argument for me, I'll let the requirements slide for this one :+1: